doc/changes/latest.inc

```diff
@@ -351,6 +351,8 @@ Bug
 
 - Fix bug with :func:`mne.events_from_annotations(raw.annotations) <mne.events_from_annotations>` when ``orig_time`` of annotations is None and ``raw.first_time > 0``, by `Alex Gramfort`_
 
+- Fix bug with :func:`mne.chpi._setup_hpi_amplitude_fitting` to improve auto time window by `Steven Bierer`_
```

Comment:
we don't document the fixes at the level of private functions. You should list the public functions that are impacted by the fix.

Reply:
I can work with @larsoner to devise a test with a range of HPI frequencies.